cosmwasm/contracts/dist/src/contract.rs

```diff
@@ -0,0 +1,59 @@
+use cosmwasm_std::{
+    generic_err, log, to_binary, unauthorized, Api, Binary, Coin, CosmosMsg, DistQuery, Env,
+    Extern, GovQuery, HandleResponse, HumanAddr, InitResponse, InitResult, Querier,
+    RewardsResponse, StdResult, Storage, VoteOption,
+};
+
+use crate::msg::{HandleMsg, InitMsg};
+
+pub fn init<S: Storage, A: Api, Q: Querier>(
+    _deps: &mut Extern<S, A, Q>,
+    _env: Env,
+    _msg: InitMsg,
+) -> InitResult {
+    Ok(InitResponse::default())
+}
+
+pub fn handle<S: Storage, A: Api, Q: Querier>(
+    deps: &mut Extern<S, A, Q>,
+    env: Env,
+    msg: HandleMsg,
+) -> StdResult<HandleResponse> {
+    match msg {
+        HandleMsg::Rewards { address } => try_query_rewards(deps, env, address),
+    }
+}
+
+pub fn try_query_rewards<S: Storage, A: Api, Q: Querier>(
+    deps: &mut Extern<S, A, Q>,
+    env: Env,
+    address: HumanAddr,
+) -> StdResult<HandleResponse> {
+    let query = DistQuery::Rewards {
+        delegator: address.clone(),
+    };
+
+    let mut query_rewards: RewardsResponse =
+        deps.querier
+            .query(&query.into())
+            .unwrap_or_else(|_| RewardsResponse {
+                rewards: vec![],
+                total: vec![],
+            });
```

Comment:
You can probably slap a `derive(Default)` on `RewardsResponse` and then call `RewardsResponse::default()` here.

EDIT: even better, use [unwrap_or_default](https://doc.rust-lang.org/std/result/enum.Result.html#method.unwrap_or_default).

Reply:
This is just some stupid test I hacked together - not going to change it